cortex-python-lib-examples/gcs-connection-skill/skills/gcs-connection-reader/main.py

```diff
@@ -0,0 +1,64 @@
+import os
```

Comment:
we prob need to add a [license](https://github.com/CognitiveScale/cortex-fabric-examples/blob/9c29b0f2ff192f5812e04238b7b7037ecfaf7198/cortex-python-lib-examples/connection-example/skills/connection-daemon/actions/connection-daemon/main.py#L1-L5)